README.md

```diff
@@ -89,3 +89,7 @@ spec:
 ```
 
 - Delete the `ibm-management-ingress-operator-xxxx` pod if you don't see any `management-ingress-xxx` pod in the `ibm-common-services` namespace
+
+## SecurityContextConstraints Requirements
+
+The IAM operator service supports running under the OpenShift Container Platform default restricted security context constraints.
```

Comment:
@rashmi43  We don't support running with default restricuted security context constraints because of the journald hostath. `auth-idp`, `auth-pap` and `auth-pdp` wont' run under the default restricted scc.

Reply:
we need this for cv-lint